src/test/java/StringTest.java

```diff
@@ -0,0 +1,53 @@
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+public class StringTest {
+
+    @Test
+    @DisplayName("1, 2를 ,로 split하면 1과 2로 분리된다.")
+    void splitOneAndTwoWithComma() {
+        String value = "1,2";
+        String[] values = value.split(",");
+
+
+        assertThat(values).contains("1", "2");

```

Comment:
```suggestion
        String[] values = value.split(",");

        assertThat(values).containsExactly("1", "2");

```
불필요한 개행이라고 생각해요
사소하지만 지켜주는 것이 좋을 것 같습니다!